### PR TITLE
Fix ob-julia when the `char` is `nil`.

### PR DESCRIPTION
**Pull request by MasonProtter**
Should fix https://github.com/gcv/julia-snail/issues/99

### CODE REVIEW

File: extensions/ob-julia/ob-julia.el [hunk 1]
```diff
@@ -76,6 +76,10 @@ your org notebook"
     ;;     (julia-snail) t))
     (julia-snail--send-to-server :Main text :async nil)))
 
+(defun julia-snail/ob-julia--maybe-goto-char (char)
+  (if char
```
Comment (gcv) [inline extensions/ob-julia/ob-julia.el:80]: Looks like this should be a `when`.

**Review by gcv — CHANGES_REQUESTED** [review]
Just a minor nitpick here.

File: extensions/ob-julia/ob-julia.el (hunk 1 above)
Comment (MasonProtter) [reply extensions/ob-julia/ob-julia.el]: Ah okay, I've switched it to `when`.

Comment (gcv) [reply extensions/ob-julia/ob-julia.el]: Thanks!

**Pull request merged**